src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -30,20 +23,6 @@ public DynamoAnalyticsSession()
 
         public void Start(DynamoModel model)
         {
-            //Whether enabled or not, we still record the startup.
```

Comment:
Moved this tracker registration/cleanup code in the Client instance. (to have access to client's data)